tests/www/test_utils.py

```diff
@@ -143,7 +143,7 @@ def some_func():
                 self.assertEqual(fake_username, kwargs['owner'])
                 mocked_session_instance.add.assert_called_once()
 
-    @mock.patch("flask_login._get_user")
+    @mock.patch("flask_login.utils._get_user")
```

Comment:
Thanks -I see what I did wrong and came to fix it to find you'd done it already :)